internal/config/config.go

```diff
@@ -31,20 +32,24 @@ type NeutronQueryRelayerConfig struct {
 
 const EnvPrefix string = "RELAYER"
 
+// NeutronChainConfig TODO: research if HomeDir parameter is needed for something but keys (so it might be optional and ignored for memory keyring)
 type NeutronChainConfig struct {
-	RPCAddr        string        `required:"true" split_words:"true"`
-	RESTAddr       string        `required:"true" split_words:"true"`
-	HomeDir        string        `required:"true" split_words:"true"`
-	SignKeyName    string        `required:"true" split_words:"true"`
-	Timeout        time.Duration `split_words:"true" default:"10s"`
-	GasPrices      string        `required:"true" split_words:"true"`
-	GasLimit       uint64        `split_words:"true" default:"0"`
-	GasAdjustment  float64       `required:"true" split_words:"true"`
-	ConnectionID   string        `required:"true" split_words:"true"`
-	Debug          bool          `split_words:"true" default:"false"`
-	KeyringBackend string        `required:"true" split_words:"true"`
-	OutputFormat   string        `split_words:"true" default:"json"`
-	SignModeStr    string        `split_words:"true" default:"direct"`
+	RPCAddr         string        `required:"true" split_words:"true"`
+	RESTAddr        string        `required:"true" split_words:"true"`
+	HomeDir         string        `required:"true" split_words:"true"`
+	SignKeyName     string        `split_words:"true"`
+	SignKeySeed     string        `split_words:"true"`
```

Comment:
The config is getting bigger and bigger, how do you feel to move keyring related fields to a standalone struct, or at least place the similar fields next to each other.

Reply:
Yup, makes sense, did it